provider/src/index.ts

```diff
@@ -21,10 +21,12 @@ export default {
     })
 
     try {
-      return handler(request, env)
+      await handler(request, env)
     } catch (error) {
       sentry.captureException(error)
-      throw error // thow again
+      return new Response('Something went wrong! Team has been notified.', {
```

Comment:
Still more accurate error alongside would be preferable so someone could notify us immediately.